### PR TITLE
docs(protocol): create mainnet-contract-logs.md

### PR DESCRIPTION
**Pull request by dantaik**

### CODE REVIEW

File: packages/protocol/deployments/mainnet-deploy-logs.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+- logs:
+  - deployed on May 1, 2024 at commit `56dddf2b6`
+
+### taiko_token
+
+- proxy: `0x10dea67478c5F8C5E2D90e5E9B26dBe60c54d800`
+- impl: `0x9ae1a067f9655dd0511390e3d70bb25933ae61eb`
+- owner: `labs.taiko.eth`
+- logs:
+  - deployed on April, 2024 at commit `?????`
```
Comment (dantaik) [inline packages/protocol/deployments/mainnet-deploy-logs.md:22]: @davidtaikocha do you still remember the commit for Taiko Token contract deployed?

Comment (dantaik) [reply packages/protocol/deployments/mainnet-deploy-logs.md]: Ideally we also create and maintain such a deploy/upgrade log for hekla.

File: packages/protocol/deployments/mainnet-contract-logs.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+- owner: `0x56706F118e42AE069F20c5636141B844D1324AE1`
+- pendingOwner: `admin.taiko.eth`
+- todo:
+  - admin.taiko.eth to accept the owner
+- logs:
+  - deployed on May 1, 2024 at commit `56dddf2b6`
+
+### bridge
+
+- proxy: `0xd60247c6848B7Ca29eDdF63AA924E53dB6Ddd8EC`
```
[the remark this answers is not in the document]
Comment (davidtaikocha) [reply packages/protocol/deployments/mainnet-contract-logs.md:42]: Right

File: packages/protocol/deployments/mainnet-contract-logs.md [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+  - deployed on May 1, 2024 at commit `56dddf2b6`
+
+### taiko_token
+
+- ens: `token.taiko.eth`
+- proxy: `0x10dea67478c5F8C5E2D90e5E9B26dBe60c54d800`
+- impl: `0x9ae1a067f9655dd0511390e3d70bb25933ae61eb`
+- owner: `labs.taiko.eth`
+- logs:
+  - deployed on April 25, 2024 at commit `?????`
```
Comment (davidtaikocha) [inline packages/protocol/deployments/mainnet-contract-logs.md:27]: After comparing the verified source code, its on: `2f6d3c62e`

File: packages/protocol/deployments/mainnet-deploy-logs.md (hunk 1 above)
Comment (davidtaikocha) [reply packages/protocol/deployments/mainnet-deploy-logs.md]: After comparing the verified source code, its on: `2f6d3c62e`

File: packages/protocol/deployments/mainnet-contract-logs.md (hunk 3 above)
Comment (dantaik) [inline packages/protocol/deployments/mainnet-contract-logs.md:27]:
```suggestion
  - deployed on April 25, 2024 at commit `2f6d3c62e`
```

**Review by davidtaikocha — APPROVED** [review]

**Review by adaki2004 — APPROVED** [review]

**Pull request merged**